### DISCUSSION TITLE
script throws segmentation fault ... not clear why

### DISCUSSION THREAD

**Discussion by ra-esmith** [google/or-tools#3401] [Routing (and legacy CP) questions]
trying to make test more simple will post again when have smaller script

**Comment by ra-esmith** [comment]
in case someone else sees this.  I had a time dimension and the "slack var" at the end of my route.  And I tried to read the slack var which was 0 (or perhaps not there).  So as soon as I stopped doing that, no segmentation fault.  Makes sense as in my case the window was the last cities/location I was visiting which clearly had 0 slack.

Evan

**Reply by Mizux** [reply]
Just to be sure, there is no slackVar associated to any vehicle's end node so it you try to read it, unexpected behaviour (often crash) will happen...

**Comment by ra-esmith** [comment]
yep!
